tests/testthat/test_recordingEndSleepHour.R

```diff
@@ -26,17 +27,24 @@ test_that("recordingEndSleepHour works as expected", {
   expect_warning( # warning from using both expand_tail_max_hour and recordingEndSleepHour
     GGIR(mode = 1, datadir = fn, outputdir = getwd(), 
          studyname = "test", overwrite = TRUE, 
-         expand_tail_max_hours = 5, recordingEndSleepHour = 8,
+         expand_tail_max_hours = 5, recordingEndSleepHour = 20,
          visualreport = FALSE, do.report = c())
   )
   
-  expect_warning( # warning from recordingEndSleepHour being too early
+  expect_error( # warning from recordingEndSleepHour being too early
     # this should trigger data expansion
     GGIR(datadir = fn, outputdir = getwd(),
          studyname = "test", overwrite = TRUE, 
          recordingEndSleepHour = 6,
          minimum_MM_length.part5 = 15)
   )
```

Comment:
I think we should rewrite the comment (lines 34-35) to make it consistent with the approach, something like 
# error from recordingEndSleepHour being too early, this should not produce any output